Add FAQ to webhook doc page, clarify some (#2525)
FusionAuth/fusionauth-site: site/docs/v1/tech/events-webhooks/index.adoc

Here are the topics in this section:

* link:/docs/v1/tech/events-webhooks/writing-a-webhook[Writing a Webhook] - Covers how to write a Webhook that will process events from FusionAuth.
* link:/docs/v1/tech/events-webhooks/securing[Securing a Webhook] - Covers how to ensure your webhooks are secured properly.

> * link:/docs/v1/tech/events-webhooks/events/[Events] - Covers all of the event types that FusionAuth sends to Webhooks

Reviewer: Assume you moved this down to bring it in line with the nav?

Author: Yeah.

Continue reading below to see how the events and webhooks are configured using the FusionAuth user interface.

* <<Tenant Settings>>
* <<Add Webhook>>
* <<Test a Webhook>>
* <<FAQs>>

== Tenant Settings

To prepare to consume FusionAuth events you'll first need to enable the event and select a transaction level that is compatible with your requirements in the tenant. 

[NOTE]
====

> You do not need to configure the tenant settings for any system level webhooks, such as the `Audit Log Create`.

Reviewer: ```suggestion
You do not need to configure the tenant settings for any system level webhooks, such as the `Audit Log Create` webhook.
```

Author: Noted your suggestion and called it 'event' for clarity.

+

> Most events are considered tenant specific. Selecting one more more tenants will cause FusionAuth to send events only for those tenants.

Reviewer: ```suggestion
Most events are scoped to a tenant. Selecting one more more tenants will cause FusionAuth to send events only for those tenants.
```

The exceptions to this are the following system events which are not tenant specific:

* `audit-log.create`
* `event-log.create`
* `kickstart.success`

> These events are configured at the system level and cannot be limited to a certain tenant.

Reviewer: ```suggestion
These events are configured at the system level and cannot be scoped to a certain tenant.
```

Event::
The JSON event to send to the webhook. This is a generated example and it may be modified before sending to replicate a specific scenario.

== FAQs

> **Q:** I have successfully tested my Webhook, but why am I not recieving specific live events? +

Reviewer: ```suggestion
**Q:** I have successfully tested my Webhook, but why am I not receiving specific live events? +
```

> **A:** There are several configurations that need to be enabled to ensure events get delivered:

Reviewer: ```suggestion
**A:** In order to receive events to your webhook endpoint, ensure the following items are configured:
```

> * Navigate to [breadcrumb]#Tenants -> Webhooks# to enable the specific events

Reviewer: ```suggestion
* Set up the Tenant events. Navigate to [breadcrumb]#Tenants -> Webhooks# and enable the specific events.
```

> * Navigate to [breadcrumb]#Settings -> Webhooks -> Tenants# and ensure the Webhook is configured for *All tenants* or checked for the desired tenant

Reviewer: ```suggestion
* Set up Webhook tenants. Navigate to [breadcrumb]#Settings -> Webhooks -> Tenants# and ensure the Webhook is configured either for [uielement]#All tenants# or your desired tenant or tenants.
```

> * Navigate to [breadcrumb]#Settings -> Webhooks -> Events# and ensure the specific events are enabled for the Webhook

Reviewer: ```suggestion
* Set up the Webhook events. Navigate to [breadcrumb]#Settings -> Webhooks -> Events# and ensure the specific events are enabled for the Webhook.
```

> 

Reviewer: ```suggestion

Unless all three of these configurations are correct, you won't receive events to your configured endpoint.

```
